flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/CodeAnalyzer.java

```diff
@@ -173,6 +174,9 @@ class CodeAnalyzer extends NodeVisitor {
     private final boolean forceAssign;
     private final String connectionScope;
 
+    private static final String LLM_CALL = "LlmCall";
```

Comment:
@MaryamZi can you confirm names ?

Reply:
Yeah, these are what we are using
- https://github.com/ballerina-platform/module-ballerinax-np/blob/dev/ballerina/api.bal#L58 
- https://github.com/ballerina-platform/module-ballerinax-np/blob/dev/ballerina/api.bal#L45